src/lib/etl-glue.ts

```diff
@@ -187,7 +189,7 @@ export class ETLByGlue extends Construct {
         '--database': transactionDatabase.databaseName,
         '--transaction_table': transactionTable.tableName,
         '--identity_table': identityTable.tableName,
-        '--id_cols': 'card1,card2,card3,card4,card5,card6,ProductCD,addr1,addr2,P_emaildomain,R_emaildomain',
+        '--id_cols': this.preprocessingJob_id_cols,
         '--cat_cols': 'M1,M2,M3,M4,M5,M6,M7,M8,M9',
```

Comment:
also extract this variable to parent stack. it is probably used by inference method